src/ethereum_test_specs/blockchain.py

```diff
@@ -300,6 +300,7 @@ class BlockchainTest(BaseTest):
     genesis_environment: Environment = Field(default_factory=Environment)
     verify_sync: bool = False
     chain_id: int = 1
+    exclude_post_state: bool = False
```

Comment:
```suggestion
    exclude_post_state: bool = False
    """
    Exclude the post state from the fixture output.
    
    In this case, the state verification is only performed based on the state root.
    """
```
Or similar to keep track of the usage of this field.